modules/gdnavigation/rvo_agent.cpp

```diff
@@ -70,7 +70,7 @@ void RvoAgent::dispatch_callback() {
 	}
 	Object *obj = ObjectDB::get_instance(callback.id);
 	if (obj == nullptr) {
-		callback.id = ObjectID();
+		return;
```

Comment:
Return here is fine, but you also need `callback.id = ObjectID();` to reset the id. In this way the next frame, the above check pass and it returns earlier.